### PR TITLE
xfconf module utils: providing a cmd_runner object

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Create a new `module_utils/xfconf.py`.
Changed `xfconf` and `xfconf_info` to use that runner instead of the features in `CmdStateModuleHelper`

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
plugins/module_utils/xfconf.py
plugins/modules/system/xfconf.py
plugins/modules/system/xfconf_info.py

### CODE REVIEW

**Review by felixfontein — APPROVED** [review]

**Comment by russoz** [comment]
Please wait until #4777 is merged so I can add that improvement here as well.

**Comment by felixfontein** [comment]
@russoz please add the improvement here as well, since that PR has been merged :)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for this improvement!

**Pull request merged**
